zeppelin-web/src/components/noteName-import/notenameImport.controller.js

```diff
@@ -19,6 +19,13 @@ angular.module('zeppelinWebApp').controller('NoteImportCtrl', function($scope, $
   $scope.note = {};
   $scope.note.step1 = true;
   $scope.note.step2 = false;
+  $scope.config = {};
+  $scope.config.maxLimit = '';
```

Comment:
I'm not sure having maxLimit in a config object is really necessary.
Could we just have $scope.maxLimit?